debian/rules

```diff
@@ -16,6 +16,7 @@ include /usr/share/dpkg/default.mk
 # package maintainers to append LDFLAGS
 #export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
+version := $(shell dpkg-parsechangelog | grep '^Version:' | cut -d' ' -f2 | cut -d- -f1)
```

Comment:
Do Debian guidelines allow using `awk` here? Because then it would be just one command instead of `grep` and two `cut`'s:

    awk -F'[ -]' '/^Version:/ {print $2}'

Or would grep and cut just be easier to understand?

Reply:
I've brought the line in from other packages, I suspect awk would work just fine though.

Reply:
Ah, alright. Could you please change it to `awk`, then, if there's nothing stopping us? Thank you.